### PR TITLE
Home task: Lesson 'Git'

### PR DESCRIPTION
**Pull request by ZikFred**
Was problems with push on fork my repo, conflict branches.. I worked on master branch in local repo(on PC), when I start did homtask do not change branch , but hope I have success)

### CODE REVIEW

**Comment by ZikFred** [comment]
I hope coped..

**Comment by ZikFred** [comment]
I did something, but I do not know what.
I hope that is true.

File: emarchik/02_GIT.me [hunk 1]
```diff
@@ -0,0 +1,45 @@
+  228  git --version
```
Comment (pluhin) [inline emarchik/02_GIT.me]: Could you please put this output into code section

Comment (ZikFred) [reply emarchik/02_GIT.me]: I don't understand what I must do.

Comment (ZikFred) [reply emarchik/02_GIT.me]: Done.
In addition, I changed the file extension.

**Comment by ZikFred** [comment]
So, home task done or no?

**Pull request merged**